tests/e2e_tests/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": ["eslint:recommended", "plugin:@typescript-eslint/eslint-recommended", "plugin:@typescript-eslint/recommended"],
+  "rules": {
+    "prettier/prettier": 2,
+    "consistent-this": ["error", "self"],
+    "import/no-named-as-default": 0,
+    "no-console": 0,
+    "no-multiple-empty-lines": 2,
+    "no-prototype-builtins": 0,
```

Comment:
dont you think we should have it?

Reply:
hmm nah, I find the readability of the alternative too bad to justify the rule 🙃